scenes/SceneEditAccount.js

```diff
@@ -52,12 +52,13 @@ export default class SceneEditAccount extends React.Component {
     allow_encrypted_data_storage: this.props.viewer.allow_encrypted_data_storage,
     changingPassword: false,
     changingAvatar: false,
+    savingNameBio: false,
     changingFilecoin: false,
     tab: 0,
   };
 
   _handleUpload = async (e) => {
-    this.setState({ changingAvatar: true });
+    this.setState({ changingAvatar: true, changingBio: true });
```

Comment:
What is changingBio doing here?

Reply:
oh whoops didn't catch that, I'll get rid of it